dev/breeze/tests/test_selective_checks.py

```diff
@@ -369,6 +369,36 @@ def assert_outputs_are_printed(expected_outputs: dict[str, str], stderr: str):
                 id="Selected Providers and docs should run",
             )
         ),
+        (
+            pytest.param(
+                ("providers/tests/system/apache/beam/file.py",),
+                {
+                    "selected-providers-list-as-string": "apache.beam common.compat google",
+                    "all-python-versions": "['3.9']",
+                    "all-python-versions-list-as-string": "3.9",
+                    "python-versions": "['3.9']",
+                    "python-versions-list-as-string": "3.9",
+                    "ci-image-build": "true",
+                    "prod-image-build": "false",
+                    "needs-helm-tests": "false",
+                    "run-tests": "true",
+                    "run-amazon-tests": "false",
+                    "docs-build": "false",
+                    "skip-pre-commits": (
+                        "identity,lint-helm-chart,mypy-airflow,mypy-dev,mypy-docs"
+                        ",mypy-providers,mypy-task-sdk,ts-compile-format-lint-ui,ts-compile-format-lint-www"
+                    ),
+                    "run-kubernetes-tests": "false",
+                    "upgrade-to-newer-dependencies": "false",
+                    "core-test-types-list-as-string": "Always",
+                    "providers-test-types-list-as-string": "Providers[apache.beam,common.compat] Providers[google]",
+                    "individual-providers-test-types-list-as-string": "Providers[apache.beam] Providers[common.compat] Providers[google]",
+                    "needs-mypy": "true",
+                    "mypy-checks": "['mypy-providers']",
+                },
+                id="Selected Providers and docs should run when system tests are modified",
```

Comment:
should we add another scenario, when both test and system changes? 
```
(
   "providers/tests/system/apache/beam/file.py",
   "providers/tests/apache/beam/file.py"
)
```

Reply:
We could, that would not make a difference, but yeah - worth adding it

Reply:
Added two more tests: 

* both tests and system tests modified
* both tests and system tests modified for more than one provider.